docs/UserGuide.adoc

```diff
@@ -118,9 +124,42 @@ Returns `john` and `John Doe`
 * `find Betsy Tim John` +
 Returns any person having names `Betsy`, `Tim`, or `John`
 
+=== Locating persons taking that module: `module`
+
+Finds persons who is taking the module as specified. +
+Format: `module MODULE_CODE`
+
+****
+* The search is case insensitive. e.g `CS2010` will match `cs2010`
+* Only full module codes will be matched e.g. `2010` will not match `CS2010`
+* Persons taking the module will be returned.
+****
+
+Examples:
+
+* `module CS2010` +
+Returns all persons having CS2010 in their modules
+
+=== Viewing common time slots in timetable: `free`
+
+Displays the common free time of two people in the addressbook. +
+Format: `free [PERSON1] [PERSON2]`
+
+****
+* Only full names will be matched e.g. `Han` will not match `Han Tan`
+* Overlap of the two persons' timetables will be displayed.
+* Auto-fill for person's name
+****
+
+Examples:
+
+* `module CS2010` +
+Returns all persons having CS2010 in their modules
```

Comment:
The examples given here are wrong. These are examples copied over from the `module` command.